packages/backend/src/peripherals/database/WithdrawableAssetRepository.ts

```diff
@@ -63,6 +66,19 @@ export class WithdrawableAssetRepository extends BaseRepository {
     return results[0]!.id
   }
 
+  async getWithdrawableAssetsByStarkKey(
```

Comment:
I think it should be called getByStarkKey, it is already in WithdrawableAssetsRepository